spec/rails_helper.rb

```diff
@@ -16,7 +16,7 @@
 
 require "rspec/rails"
 require Rails.root.join("spec", "shared_specs.rb")
-Dir[Rails.root.join("spec", "support", "**", "*.rb")].sort.each { |file| require file }
+Dir[Rails.root.join("spec", "support", "**", "*.rb"), Rails.root.join("db", "migrate", "**", "*.rb")].sort.each { |file| require file }
```

Comment:
Is this leftover from testing the migrate script without the service class? delete?